### ISSUE TITLE
Object 'MR' has several compatible declarations in modules MyLib_iosArm64, MyLib_iosX64Main

### ISSUE THREAD

**Issue by xrofa** [icerockdev/moko-resources#58]
I'm using the latest version of the lib "0.9.1".

My build gradle is as follows:
```
kotlin {
    iosX64() {
        binaries.framework {
            baseName = "MyLib"
            isStatic = false
        }
    }
    iosArm64() {
        binaries.framework {
            baseName = "MyLib"
            isStatic = false
        }
    }
    sourceSets {
        ...
        val iosX64Main by getting
        val iosX64Test by getting
        val iosArm64Main by getting {
            dependsOn(iosX64Main)
        }
        val iosArm64Test by getting {
            dependsOn(iosX64Test)
        }

        configure(
            listOf(
                iosArm64Main,
                iosX64Main
            )
        ) {
            dependencies {
                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-native:$coroutines_version")
                implementation("org.jetbrains.kotlinx:kotlinx-serialization-runtime-native:$serialization_version")
                implementation("org.jetbrains.kotlinx:kotlinx-serialization-properties-native:$serialization_version")
                implementation("io.ktor:ktor-client-ios:$ktor_version")
                implementation("io.ktor:ktor-client-core-native:$ktor_version")
                implementation("io.ktor:ktor-client-serialization-native:$ktor_version")
            }
        }
    }
}
```

After that I updated to (the target def remains the same, only renamed the folder to iosMain from iosX64Main):
```
val iosMain by creating {
            dependencies {
                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-native:$coroutines_version")
                implementation("org.jetbrains.kotlinx:kotlinx-serialization-runtime-native:$serialization_version")
                implementation("org.jetbrains.kotlinx:kotlinx-serialization-properties-native:$serialization_version")
                implementation("io.ktor:ktor-client-ios:$ktor_version")
                implementation("io.ktor:ktor-client-core-native:$ktor_version")
                implementation("io.ktor:ktor-client-serialization-native:$ktor_version")
            }
        }

        val iosX64Main by getting {
            dependsOn(iosMain)
        }

        val iosArm64Main by getting {
            dependsOn(iosMain)
        }
        val iosX64Test by getting
        val iosArm64Test by getting
```

And now when building I'm getting: 
`Expected object 'MR' has no actual declaration in module <MyLib_test> for Native`

**Comment by Alex009** [comment]
hi! thanks for report. i will fix this bug soon

**Comment by Alex009** [comment]
fix merged in develop. will be released in 0.10.0
